db/team.go

```diff
@@ -0,0 +1,61 @@
+package db
+
+import (
+	"log"
+	"os"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/aws/aws-sdk-go/service/dynamodb/dynamodbattribute"
+	"github.com/fatih/structs"
+)
+
+type Team struct {
+	SlackTeamId            string `json:"slack_team_id"`
+	SlackBotUserToken      string `json:"slack_bot_user_oauth_token"`
+	SlackRegularOauthToken string `json:"slack_regular_oauth_token"`
+}
+
+func FindOrCreateTeamById(id string) (*Team, error) {
+	tableName := os.Getenv("DYNAMODB_TABLE_PREFIX") + "teams"
+	// CreateTableIfNotCreated(tableName, "slack_team_id")
+
+	team, err := FindTeamById(id)
+	if err != nil {
+		if err != nil {
```

Comment:
same as line 24?

Reply:
AH yes, I wanted to catch only the table already exists error, I need to copy paste your code from somewhere